CrudMicroservice.Data/Entities/CarAccident.cs

```diff
@@ -0,0 +1,69 @@
+﻿using MongoDbGenericRepository.Attributes;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace CrudMicroservice.Data.Entities
+{
+    [CollectionName("CarAccidents")]
+    public class CarAccident
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+
+        [BsonElement("SerialNumber")]
+        [JsonProperty("SerialNumber")]
+        public string SerialNumber { get; set; }
+
+        [BsonElement("InspectorId")]
+        [JsonProperty("InspectorId")]
+        public string InspectorId { get; set; }
+
+        [BsonElement("RegistrationDateTime")]
+        [JsonProperty("RegistrationDateTime")]
+        public DateTime RegistrationDateTime { get; set; }
+
+        [BsonElement("AddressOfAccident")]
+        [JsonProperty("AddressOfAccident")]
+        public AddressOfAccident Address { get; set; }
+
+        [BsonElement("SideOfAccident")]
+        [JsonProperty("SideOfAccident")]
+        public Side SideOfAccident { get; set; }
```

Comment:
I think that can be many participants of accident, so we should store list of sides

Reply:
no - for each participant of car accident issued separate protocol